cluster/manifests/01-admission-control/teapot.yaml

```diff
@@ -132,7 +132,7 @@ webhooks:
     rules:
       - operations: ["CREATE", "UPDATE"]
         apiGroups: ["extensions", "networking.k8s.io"]
-        apiVersions: ["v1beta1"]
+        apiVersions: ["v1"]
```

Comment:
This change also needs a change to the admission-controller code because we currently hardcode `v1beta1` only. We should also remove `extentions` from the `apiGroups` above as there is not v1 `extensions`.